rogue-thi-app/pages/index.js

```diff
@@ -161,260 +147,33 @@ const ALL_DASHBOARD_CARDS = [
-    localStorage.personalizedDashboard = JSON.stringify(entries.map(x => x.key))
-    localStorage.personalizedDashboardHidden = JSON.stringify(hiddenEntries.map(x => x.key))
-    setShownDashboardEntries(entries)
-    setHiddenDashboardEntries(hiddenEntries)
-  }
-
-  /**
-   * Moves a dashboard entry to a new position.
-   * @param {number} oldIndex Old position
-   * @param {number} diff New position relative to the old position
-   */
-  function moveDashboardEntry (oldIndex, diff) {
-    const newIndex = oldIndex + diff
-    if (newIndex < 0 || newIndex >= shownDashboardEntries.length) {
-      return
-    }
-
-    const entries = shownDashboardEntries.slice(0)
-    const entry = entries[oldIndex]
-    entries.splice(oldIndex, 1)
-    entries.splice(newIndex, 0, entry)
-
-    changeDashboardEntries(entries, hiddenDashboardEntries)
-  }
-
-  /**
-   * Hides a dashboard entry.
-   * @param {string} key Entry key
-   */
-  function hideDashboardEntry (key) {
-    const entries = shownDashboardEntries.slice(0)
-    const hiddenEntries = hiddenDashboardEntries.slice(0)
-
-    const index = entries.findIndex(x => x.key === key)
-    if (index >= 0) {
-      hiddenEntries.push(entries[index])
-      entries.splice(index, 1)
-    }
-
-    changeDashboardEntries(entries, hiddenEntries)
-  }
-
-  /**
-   * Unhides a dashboard entry.
-   * @param {number} index Entry position
-   */
-  function bringBackDashboardEntry (index) {
-    const entries = shownDashboardEntries.slice(0)
-    const hiddenEntries = hiddenDashboardEntries.slice(0)
-
-    entries.push(hiddenEntries[index])
-    hiddenEntries.splice(index, 1)
-
-    changeDashboardEntries(entries, hiddenEntries)
-  }
-
-  /**
-   * Resets which dashboard entries are shown and their order to default
-   */
-  function resetOrder () {
-    const defaultEntries = getDefaultDashboardOrder()
-    setShownDashboardEntries(defaultEntries.shown)
-    setHiddenDashboardEntries(defaultEntries.hidden)
-    changeDashboardEntries(defaultEntries.shown, defaultEntries.hidden)
-  }
-
-  /**
-   * Changes the current theme.
-   * @param {string} theme Theme key
-   */
-  function changeTheme (theme) {
-    localStorage.theme = theme
-    setTheme(theme)
-    setShowThemeModal(false)
-  }
+  const {
+    shownDashboardEntries,
+    hideDashboardEntry
+  } = useDashboard()
+  const [showPersonalizeModal, setShowPersonalizeModal] = useContext(ShowPersonalizeModal)
+  const router = useRouter()
 
   return (
     <AppContainer>
       <AppNavbar title="neuland.app" showBack={false}>
-        <AppNavbar.Button onClick={() => setShowThemeModal(true)}>
-          <FontAwesomeIcon title="Personalisieren" icon={faPen} fixedWidth />
+        <AppNavbar.Button onClick={() => setShowPersonalizeModal(true)}>
+          <FontAwesomeIcon icon={faPen} title={'Personalisieren'} fixedWidth />
+        </AppNavbar.Button>
+        <AppNavbar.Button onClick={() => forgetSession(router)}>
```

Comment:
Can you remove the logout button from the dashboard? I think the intention was to have it only on the profile page

Reply:
Done. Removed the Theming Button also. As the Design suggested and we discussed.